### ISSUE TITLE
Uninstall

### ISSUE THREAD

**Issue by weems** [SummitRoute/osxlockdown#44]
Is there a Undo / Uninstall Option?

Thanks

**Comment by TraderStf** [comment]
Download LockDown from https://www.objective-see.com/
You will be able to easily switch on or off the options you want.

**Comment by TX-Mac** [comment]
TraderStf - sadly, I don't believe it works that way.  LockDown from Objective-See will allow you to switch on options via the check boxes, but un-checking items will not revert the settings.  Do a quick test - toggle all of the options off and only select one, then click Fix.  Next, select all of the options and click Audit.  My guess is that you'll find, as I did, that all of the previously selected options are still enabled.

**Comment by ghost** [comment]
> TraderStf - sadly, I don't believe it works that way. LockDown from Objective-See will allow you to switch on options via the check boxes, but un-checking items will not revert the settings. Do a quick test - toggle all of the options off and only select one, then click Fix. Next, select all of the options and click Audit. My guess is that you'll find, as I did, that all of the previously selected options are still enabled.

Yes. There is check and "do" commands, there is no undo commands.

**Comment by TraderStf** [comment]
Only very few options are not available in Apple System Preferences or in other apps with a GUI.

Which one can't you undo?

**Comment by Eitot** [comment]
Perhaps each setting could have a key for the system default, so that a reversal is possible.

**Comment by matteocng** [comment]
@Eitot a reset to defaults function sounds like a nice idea. -backup and -restore flags to save / restore all the settings to / from a file could also be handy to have.

**Comment by TraderStf** [comment]
http://stackoverflow.com/questions/2346171/writing-gui-frontend-for-commandline-application-in-mac-os-x

or something newer?

this?
https://www.bluem.net/en/mac/pashua/
